app/components/trainees/confirmation/degrees/view.rb

```diff
@@ -0,0 +1,76 @@
+module Trainees
+  module Confirmation
+    module Degrees
+      class View < GovukComponent::Base
+        attr_accessor :trainee
+
+        def initialize(trainee:)
+          @trainee = trainee
```

Comment:
Does this component even need the trainee? Looks like it's just interrogating a degree object.

Reply:
🤔 Good point. I suppose at the time I was thinking just pass it the main thing i.e the trainee and let it work out which parts of it to use.

What do you reckon, drop trainee and just pass it a list of degrees?

Reply:
`trainee` is now dropped in favour of passing in a collection of degrees to the component.